tests/lib/prepare.sh

```diff
@@ -942,8 +942,10 @@ EOF
         IMAGE_CHANNEL="$GADGET_CHANNEL"
     fi
 
-    if os.query is-core20; then
-        snap download --basename=pc-kernel --channel="20/$KERNEL_CHANNEL" pc-kernel
+    if os.query is-core20 || os.query is-core22; then
+        BRANCH="${SPREAD_SYSTEM##ubuntu-core-}"
```

Comment:
you could do:
if os.query is-core20; then
  BRANCH=20
else
...

Reply:
Yes, it is a bit better to read. Done.